lib/svg-sprite.js

```diff
@@ -117,16 +117,20 @@ SVGSpriter.prototype.add = function(file, name, svg) {
             throw e;
         }
 
+        // Resolve path before splitting it into base and path
+        // so that Shape will properly extract a shape name later on.
+        file = path.resolve(file);
+
         // Instantiate a vinyl file
         file = new File({
             base: path.dirname(file),
             path: file,
             contents: Buffer.from(svg)
         });
+    } else {
```

Comment:
This branch seems to be uncovered. Are we sure it's needed? Or are the tests enough?

Reply:
It seems that the tests are enough for now

Reply:
Nope, these are still uncovered.